verify/verify.go

```diff
@@ -866,20 +866,52 @@ func verifyHash256(quote *pb.QuoteV4) error {
 	return nil
 }
 
-func isSvnHigherOrEqual(svn []byte, components []pcs.TcbComponent) bool {
-	for i := range svn {
-		if svn[i] < components[i].Svn {
+func isCPUSvnHigherOrEqual(pckCertCPUSvnComponents []byte, sgxTcbcomponents []pcs.TcbComponent) bool {
+	for i := range pckCertCPUSvnComponents {
+		if pckCertCPUSvnComponents[i] < sgxTcbcomponents[i].Svn {
 			return false
 		}
 	}
 	return true
 }
 
+func isTdxTcbSvnHigherOrEqual(teeTcbSvn []byte, tdxTcbcomponents []pcs.TcbComponent) bool {
+	start := 0
```

Comment:
Same here. Len of both arrays should be compared?

Reply:
... and that the length matches the expected size as per spec unless this is already checked while parsing.

Reply:
Already checks while parsing that TeeTcbSvn array is 16 bytes.